packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -17,13 +17,23 @@ const GuidanceWrapper = styled.div`
   height: 100%;
   position: absolute;
   background-color: ${GUIDANCE_BACKGROUND};
+  border: 1px solid transparent;
   color: ${C_WHITE};
+  @media screen and (-ms-high-contrast: active) {
+    background-color: transparent;
+  }
+`;
 
+const Content = styled.div`
+  display: flex;
+  align-items: center;
```

Comment:
Is flex box necessary here?

Reply:
Ohh didn't notice I left that in there, will remove.